compiler/noirc_frontend/src/ast/statement.rs

```diff
@@ -243,10 +243,15 @@ pub trait Recoverable {
 #[derive(Debug, PartialEq, Eq, Clone)]
 pub struct ModuleDeclaration {
     pub ident: Ident,
+    pub visibility: ItemVisibility,
 }
 
 impl std::fmt::Display for ModuleDeclaration {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        if self.visibility != ItemVisibility::Private {
+            write!(f, "{}", self.visibility)?;
+        };
```

Comment:
Ah, I see the check now. This should have a space after it though

Reply:
```suggestion
        if self.visibility != ItemVisibility::Private {
            write!(f, "{} ", self.visibility)?;
        };
```